### PR TITLE
YUD-37 - Create UI for wishlist and home screen

### PR DESCRIPTION
**Pull request by liwjln**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -71,5 +71,6 @@ dependencies {
     implementation 'com.github.Dhaval2404:ImagePicker:2.1'  // Choose a single image: https://github.com/Dhaval2404/ImagePicker
     implementation "androidx.activity:activity-ktx:1.2.3"  // Only this version works. Do not change!!!
     implementation "androidx.fragment:fragment-ktx:1.3.3"  // Only this version works. Do not change!!!
+    implementation "androidx.recyclerview:recyclerview:1.3.2"
```
Comment (ThanhVan8) [inline app/build.gradle:74]: đã có ở nhánh main

File: app/src/main/res/layout/activity_main.xml
Comment (ThanhVan8) [inline app/src/main/res/layout/activity_main.xml:1]: class MainActivity đã rename thành StudentMainActivity

**Review by ThanhVan8 — CHANGES_REQUESTED** [review]
resolve conflicts

**Comment by liwjln** [comment]
Resolve conflict in YUD-36

**Pull request closed without merging**